### ISSUE TITLE
New submissions for Tue,  5 Dec 23

### ISSUE THREAD

**Issue by olozhika** [olozhika/ArXivDaily_StarFormation#161]
# New submissions for Tue,  5 Dec 23
Auto update Star Formation & Molecular Cloud papers at about 2:30am UTC (10:30am Beijing time) every weekday.

阅读 `Usage.md`了解如何使用此repo实现个性化的Arxiv论文推送

See `Usage.md` for instructions on how to personalize the repo. 

Keyword list: ['star formation', 'molecular cloud', 'interstellar medium', 'cloud', 'clump', 'core', 'filament', 'atomic gas', 'N-PDF']

Excluded: ['galaxies', 'galaxy cluster', 'AGN']

#### Generating Images of the M87* Black Hole Using GANs
 - **Authors:** Arya Mohan, Pavlos Protopapas, Keerthi Kunnumkai, Cecilia Garraffo, Lindy Blackburn, Koushik Chatterjee, Sheperd S. Doeleman, Razieh Emami, Christian M. Fromm, Yosuke Mizuno, Angelo Ricarte
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA); Machine Learning (cs.LG); Image and Video Processing (eess.IV)
 - **Arxiv link:** https://arxiv.org/abs/2312.01005
 - **Pdf link:** https://arxiv.org/pdf/2312.01005
 - **Abstract**
 In this paper, we introduce a novel data augmentation methodology based on Conditional Progressive Generative Adversarial Networks (CPGAN) to generate diverse black hole (BH) images, accounting for variations in spin and electron temperature prescriptions. These generated images are valuable resources for training deep learning algorithms to accurately estimate black hole parameters from observational data. Our model can generate BH images for any spin value within the range of [-1, 1], given an electron temperature distribution. To validate the effectiveness of our approach, we employ a convolutional neural network to predict the BH spin using both the GRMHD images and the images generated by our proposed model. Our results demonstrate a significant performance improvement when training is conducted with the augmented dataset while testing is performed using GRMHD simulated data, as indicated by the high R2 score. Consequently, we propose that GANs can be employed as cost effective models for black hole image generation and reliably augment training datasets for other parameterization algorithms.
#### Chemistry of Complex Organic Molecules in the V883 Ori Disk Revealed by  ALMA Band 3 Observations
 - **Authors:** Yoshihide Yamato, Shota Notsu, Yuri Aikawa, Yuki Okoda, Hideko Nomura, Nami Sakai
 - **Subjects:** Earth and Planetary Astrophysics (astro-ph.EP); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2312.01300
 - **Pdf link:** https://arxiv.org/pdf/2312.01300
 - **Abstract**
 Complex organic molecules (COMs) in protoplanetary disks are key to understanding the origin of volatiles in comets in our solar system, yet the chemistry of COMs in protoplanetary disks remains poorly understood. Here we present Atacama Large Millimeter/submillimeter Array (ALMA) Band 3 observations of the disk around the young outbursting star V883 Ori, where the COMs sublimate from ices and are thus observable thanks to the warm condition of the disk. We have robustly identified ten oxygen-bearing COMs including $^{13}$C-isotopologues in the disk-integrated spectra. The radial distributions of the COM emission, revealed by the detailed analyses of the line profiles, show the inner emission cavity, similar to the previous observations in Band 6 and Band 7. We found that the COMs abundance ratios with respect to methanol are significantly higher than those in the warm protostellar envelopes of IRAS 16293-2422 and similar to the ratios in the solar system comet 67P/Churyumov-Gerasimenko, suggesting the efficient (re-)formation of COMs in protoplanetary disks. We also constrained the $^{12}$C/$^{13}$C and D/H ratios of COMs in protoplanetary disks for the first time. The $^{12}$C/$^{13}$C ratios of acetaldehyde, methyl formate, and dimethyl ether are consistently lower ($\sim$ 20-30) than the canonical ratio in the interstellar medium ($\sim$ 69), indicating the efficient $^{13}$C-fractionation of CO. The D/H ratios of methyl formate are slightly lower than the values in IRAS 16293-2422, possibly pointing to the destruction and reformation of COMs in disks. We also discuss the implications for nitrogen and sulfur chemistry in protoplanetary disks.
#### Two long-period giant planets around two giant stars: HD 112570 and HD  154391
 - **Authors:** Guang-Yao Xiao, Huan-Yu Teng, Jianzhao Zhou, Bun'ei Sato, Yu-Juan Liu, Shaolan Bi, Takuya Takarada, Masayuki Kuzuhara, Marc Hon, Liang Wang, Masashi Omiya, Hiroki Harakawa, Fei Zhao, Gang Zhao, Eiji Kambe, Hideyuki Izumiura, Hiroyasu Ando, Kunio Noguchi, Wei Wang, Meng Zhai, Nan Song, Chengqun Yang, Tanda Li, Timothy D. Brandt, Michitoshi Yoshida, Yoichi Itoh, Eiichiro Kokubo
 - **Subjects:** Earth and Planetary Astrophysics (astro-ph.EP)
 - **Arxiv link:** https://arxiv.org/abs/2312.01388
 - **Pdf link:** https://arxiv.org/pdf/2312.01388
 - **Abstract**
 We present the discoveries of two giant planets orbiting the red giant branch (RGB) star HD 112570 and the red clump (RC) star HD 154391, based on the radial velocity (RV) measurements from Xinglong station and Okayama Astrophysical Observatory (OAO). Spectroscopic and asteroseismic analyses suggest that HD 112570 has a mass of $1.15\pm0.12\,M_{\odot}$, a radius of $9.85\pm0.23\,R_{\odot}$, a metallicity [Fe/H] of $-0.46\pm0.1$ and a ${\rm log}\,g$ of $2.47\pm0.1$. With the joint analysis of RV and Hipparcos-Gaia astrometry, we obtain a dynamical mass of $M_{\rm p}={3.42}_{-0.84}^{+1.4}\ M_{\rm Jup}$, a period of $P={2615}_{-77}^{+85}$ days and a moderate eccentricity of $e={0.20}_{-0.14}^{+0.16}$ for the Jovian planet HD 112570 b. For HD 154391, it has a mass of $2.07\pm0.03\,M_{\odot}$, a radius of $8.56\pm0.05\,R_{\odot}$, a metallicity [Fe/H] of $0.07\pm0.1$ and a ${\rm log}\,g$ of $2.86\pm0.1$. The super-Jupiter HD 154391 b has a mass of $M_{\rm p}={9.1}_{-1.9}^{+2.8}\ M_{\rm Jup}$, a period of $P={5163}_{-57}^{+60}$ days and an eccentricity of $e={0.20}_{-0.04}^{+0.04}$. We found HD 154391 b has one of the longest orbital period among those ever discovered orbiting evolved stars, which may provide a valuable case in our understanding of planetary formation at wider orbits. Moreover, while a mass gap at $4\,M_{\rm Jup}$ seems to be present in the population of giant stars, there appears to be no significant differences in the distribution of metallicity among giant planets with masses above or below this threshold. Finally, The origin of the abnormal accumulation near 2 au for planets around large evolved stars ($R_{\star}>21\,R_{\odot}$), remains unclear.
#### Feedback from protoclusters does not significantly change the kinematic  properties of the embedded dense gas structures
 - **Authors:** J. W. Zhou, S. Dib, F. Wyrowski, T. Liu, S. H. Li, P. Sanhueza, M. Juvela, F. W. Xu, H. L. Liu, T. Baug, Y. P. Peng, K. M. Menten, L. Bronfman, C. W. Lee
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2312.01497
 - **Pdf link:** https://arxiv.org/pdf/2312.01497
 - **Abstract**
 A total of 64 ATOMS sources at different evolutionary stages were selected to investigate the kinematics and dynamics of gas structures under feedback. We identified dense gas structures based on the integrated intensity map of H$^{13}$CO$^+$ J=1-0 emission, and then extracted the average spectra of all structures to investigate their velocity components and gas kinematics. For the scaling relations between velocity dispersion $\sigma$, effective radius $R$ and column density $N$ of all structures, $\sigma-N*R$ always has a stronger correlation compared to $\sigma-N$ and $\sigma-R$. There are significant correlations between velocity dispersion and column density, which may imply that the velocity dispersion originates from gravitational collapse, also revealed by the velocity gradients. The measured velocity gradients for dense gas structures in early-stage sources and late-stage sources are comparable, indicating gravitational collapse through all evolutionary stages. We quantitatively estimated the velocity dispersion generated by the outflows, inflows, ionized gas pressure and radiation pressure, and found that the ionized gas feedback is stronger than other feedback mechanisms. However, although feedback from HII regions is the strongest, it does not significantly affect the physical properties of the embedded dense gas structures. Combining with the conclusions in Zhou+2023 on cloud-clump scales, we suggest that although feedback from cloud to core scales will break up the original cloud complex, the substructures of the original complex can be reorganized into new gravitationally governed configurations around new gravitational centers. This process is accompanied by structural destruction and generation, and changes in gravitational centers, but gravitational collapse is always ongoing.
#### UV-photoprocessing of acetic acid (CH3COOH)-bearing interstellar ice  analogs
 - **Authors:** C. del Burgo Olivares, H. Carrascosa, B. Escribano, G. M. Muñoz Caro, R. Martín-Doménech
 - **Subjects:** Instrumentation and Methods for Astrophysics (astro-ph.IM); Earth and Planetary Astrophysics (astro-ph.EP); Astrophysics of Galaxies (astro-ph.GA); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2312.01845
 - **Pdf link:** https://arxiv.org/pdf/2312.01845
 - **Abstract**
 Acetic acid (CH3COOH) was detected in the gas toward interstellar clouds, hot cores, protostars and comets. Its formation in ice mantles was proposed and acetic acid awaits detection in the infrared spectra of the ice as most other COMs except methanol. The thermal annealing and UV-irradiation of acetic acid in the ice was simulated experimentally in this work under astrophysically relevant conditions. The experiments were performed under ultra-high vacuum conditions. An ice layer was formed by vapor deposition onto a cold substrate, and was warmed up or exposed to UV photons. The ice was monitored by infrared spectroscopy while the molecules desorbing to the gas phase were measured using a quadrupole mass spectrometer. The transformation of the CH3COOH monomers to cyclic dimers occurs at 120 K and the crystal form composed of chain polymers was observed above 160 K during warm-up of the ice. Ice sublimation proceeds at 189 K in our experiments. Upon UV-irradiation simpler species and radicals are formed, which lead to a residue made of complex molecules after warm-up to room temperature. The possible formation of oxalic acid needs to be confirmed. The photodestruction of acetic acid molecules is reduced when mixed with water in the ice. This work may serve to search for the acetic acid photoproducts in lines of sight where this species is detected. A comparison of the reported laboratory infrared spectra with current JWST observations allows to detect or set upper imits on the CH3COOH abundances in interstellar and circumstellar ice mantles.
#### Water content of rocky exoplanets in the habitable zone
 - **Authors:** Ádám Boldog, Vera Dobos, László L. Kiss, Marijn van der Perk, Amy C. Barr
 - **Subjects:** Earth and Planetary Astrophysics (astro-ph.EP)
 - **Arxiv link:** https://arxiv.org/abs/2312.01893
 - **Pdf link:** https://arxiv.org/pdf/2312.01893
 - **Abstract**
 In this study we investigated the interiors of rocky exoplanets in order to identify those that may have large quantities of water. We modelled the interiors of 28 rocky exoplanets, assuming four different layers - an iron core, a rock mantle, a high-pressure ice layer, and a surface ice/water layer. Due to observational bias, our study is limited to habitable zone exoplanets. We determined a range of possible water mass fractions for each planet consistent with the modelled planetary structures. We calculated the tidal heating experienced by these exoplanets through gravitational interactions with their host stars, assuming a temperature- and composition dependent Maxwell viscoelastic rheology. Assuming radioactive elemental abundances observed in Solar System meteorites, we also calculated the radiogenic heat flux inside the planets. We estimated the probability of the presence of a thick ocean layer in these planets, taking into account the effect of both tidal and radiogenic heating. Our results showed that Proxima Centauri b, Ross 128 b, Teegarden's b and c, GJ 1061 c and d, and TRAPPIST-1 e may have an extended liquid water reservoir. Furthermore, extremely high H2O-content of the exoplanets Kepler-62 f, Kepler-1652 b, Kepler-452 b, and Kepler-442 b suggests that these planets may maintain a water vapour atmosphere and may in fact be examples of larger ocean worlds. Upon the discovery of new rocky exoplanets beyond the habitable zone, our study can be extended to icy worlds.
#### The jets and the neutron star kick velocity of the supernova remnant CTB  1
 - **Authors:** Ealeal Bear, Noam Soker (Technion, Israel)
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2312.02026
 - **Pdf link:** https://arxiv.org/pdf/2312.02026
 - **Abstract**
 We identify jet-shaped morphology in the core-collapse supernova remnant (SNR) CTB 1 that includes two opposite structural features. We identify these as the imprints of a pair of jets that were among the last jets to explode the massive stellar progenitor of CTB 1. We find the projected angle between the jets' axis and the direction of the pulsar velocity, which is the neutron star natal kick, to be 78 degrees. We tentatively identify possible signatures of a second pair of opposite jets along a different direction. If holds, SNR CTB 1 has a point-symmetric structure. The morphology and large angle of the jets' axis to kick velocity are the expectations of the jittering jets explosion mechanism (JJEM) of core-collapse supernovae.
#### Analysis of Neutron Star $f-$mode Oscillations in General Relativity  with Spectral Representation of Nuclear Equations of State
 - **Authors:** Debanjan Guha Roy, Tuhin Malik, Swastik Bhattacharya, Sarmistha Banik
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE); General Relativity and Quantum Cosmology (gr-qc); Nuclear Theory (nucl-th)
 - **Arxiv link:** https://arxiv.org/abs/2312.02061
 - **Pdf link:** https://arxiv.org/pdf/2312.02061
 - **Abstract**
 We conduct a detailed analysis of quasinormal $f-$mode frequencies in neutron stars (NS), within the linearized General Relativistic formalism. From Bayesian inference, we derived approximately 9000 nuclear Equations of State (EOS) subject to various constraints including nuclear saturation properties, the pure neutron matter EOS constraint obtained within $\chi$EFT, and pQCD at densities relevant to NS cores. The composition and oscillatory dynamics of NS are then investigated using this set. The EOS are transformed into a spectral representation, aiding in the efficient computation of NS properties. The median frequency values of the $f-$mode for NS with masses ranging from 1.4$M_\odot$ to 2.0$M_{\odot}$ lie between 1.80 and 2.20 kHz for our entire EOS set. Our findings do not reveal a strong correlation between $f-$mode frequencies and individual nuclear saturation properties of the EOS. This suggests the need for more complex methods to unravel multiple-parameter relationships. We noticed a strong relationship between the radii and $f-$mode frequencies for different NS masses. Using this correlation along with NICER observations of PSR J0740+6620 and PSR 0030+0451, we obtained constraints that have minimal overlap in the radius domain and differ in the frequency domain from our entire nucleonic EOS set. This indicates that there may be a need to consider additional exotic particles or maybe a deconfined quark phase in the EOS relevant to the NS core. We argue that future observations of the radius or $f-$mode frequency for more than one NS mass, particularly at the extremes, are likely to settle the issue by either ruling out only nucleonic EOS or providing definitive evidence in its favour.

by olozhika (Xing Yuchen). 

2023-12-05
